.github/workflows/pr-merge.yml

```diff
@@ -0,0 +1,19 @@
+name: PR Merged
+
+on:
+  pull_request:
+    branches:
+      - ni/priv
+      - ni/pub/*
+    types: [closed]
+
+jobs:
+  main:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ni/current
+      - run: git rebase ${{ github.ref }}
```

Comment:
Will the github action have permission to push to ni/current?

Reply:
Ok, this seems to be a common problem without a built-in solution: https://github.community/t/how-to-push-to-protected-branches-in-a-github-action/16101
Here's what I'm going to try:
I generated a personal access token using my account as a secret for the workflow to use. One annoyance is that repository secrets by default are available to all workflows, so technically someone could abuse the token just by creating a new workflow in a dev branch! I instead used ["environments"](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment) so that we can scope the secrets only to specific branches.